loader/src/sources/prepmod/index.js

```diff
@@ -41,6 +52,27 @@ async function getDataForHost(host) {
   );
 }
 
+async function getKnownLocations(state) {
+  const client = ApiClient.fromEnv();
+  const locations = await client.getLocations({
+    state,
+    provider: "prepmod",
+  });
+
+  // Create a lookup object indexed by external ID.
+  const result = new Map();
```

Comment:
Is the reason to use a `Map` here rather than a plain old object just because you wanted `.values()` below? Seems like convention elsewhere in the codebase is to call `Object.values(...)`

Reply:
No special reason, but I guess I tend towards `Map` when it isn’t especially more or less useful than a plain object, since offers the same API when you need its more special features as when your indexes are strings. I guess it’s fair to say we’ve used plain objects in a lot of other places I normally wouldn’t, so this feels out of place as a result. Can change.